applications/asset_tracker_v2/doc/app_module.rst

```diff
@@ -0,0 +1,101 @@
+.. _asset_tracker_v2_app_module:
```

Comment:
There are unrelated changes in the PR.

Reply:
This was because we were pulling in a PR that wasn't in yet